strawberryfields/ops.py

```diff
@@ -1771,7 +1771,7 @@ class GraphEmbed(Decomposition):
 
     Args:
         A (array): an :math:`N\times N` complex or real symmetric matrix
-        mean_photon_per_mode (float): guarantees that the mean photon number in the pure Gaussian state
+        mean_photon (float): guarantees that the mean photon number in the pure Gaussian state
```

Comment:
@josh146 just curious why this was changed?

Reply:
I don't remember anymore... I think I was bringing it inline with other frontend components, that all used `mean_photon` as opposed to `mean_photon_per_mode`

Reply:
From what I recall, this was specifically changed to be `mean_photon_per_mode` during the graph embed bugfix, since previously it was looking at total mean photon number, and doing something physically different

@nquesada can you comment?

Reply:
The name of the variable should be ```mean_photon_per_mode```.  The docstring and the usage when passing to ```dec.graph_embed``` are correct.